pipeline-scripts/buildlib.groovy

```diff
@@ -68,6 +68,15 @@ def registry_login() {
     }
 }
 
+def registry_quay_qe_login() {
```

Comment:
yeah, i think i'd like to have the location parameterized, just so you don't have to look at the code to see where it lands:
```suggestion
def registry_quay_qe_login(config_dir="./qe_quay_config") {
```